insights/client/auto_config.py

```diff
@@ -128,6 +132,9 @@ def _try_satellite6_configuration(config):
         rhsm_proxy_port = rhsm_config.get('server', 'proxy_port').strip()
         rhsm_proxy_user = rhsm_config.get('server', 'proxy_user').strip()
         rhsm_proxy_pass = rhsm_config.get('server', 'proxy_password').strip()
+        rhsm_no_proxy = rhsm_config.get('server', 'no_proxy').strip()
+        if rhsm_no_proxy.lower() == 'None' or rhsm_no_proxy == '':
```

Comment:
`str.lower()` is supposed to return a copy of the string converted to lowercase and you compare it to `None`.

Reply:
oh my mistake, it needs to be compare with `'none' `